dedupe/ckan_api.py

```diff
@@ -181,7 +192,7 @@ def remove_package(self, package_id):
             log.info('Not removing package in dry_run package=%s', package_id)
             return
 
-        self.request('POST', '/action/dataset_purge', json={
+        self.request('POST', '/action/package_purge', json={
```

Comment:
You didn't grab the latest main branch, otherwise this change wouldn't be in here. Try git rebase, or manually inspect the latest [PR merge](https://github.com/GSA/datagov-dedupe/pull/21/files) and pull in the changes.